storm-core/src/storm.thrift

```diff
@@ -146,6 +146,12 @@ struct TopologySummary {
 513: optional string sched_status;
 514: optional string owner;
 515: optional i32 replication_count;
+521: optional double requested_memonheap;
```

Comment:
here and elsewhere: Can we make these arguments sequential, or is there a reason we did not want to do that?